models/Asset/Folder.php

```diff
@@ -179,11 +180,18 @@ public function getPreviewImage(bool $force = false)
 
             if ($count && !$skipped) {
                 $localFile = File::getLocalTempFilePath('jpg');
-                imagejpeg($collage, $localFile, 60);
+                if (false === imagejpeg($collage, $localFile, 60)) {
+                    Logger::info('Creation of collage file of asset folder ' . $this->getRealFullPath() . ' is failed.');
 
-                if (filesize($localFile) > 0) {
```

Comment:
this check would be missing, it seems would "revert' this fix in https://github.com/pimcore/pimcore/pull/11722
![image](https://github.com/user-attachments/assets/0231b6e4-2178-49a7-b65f-3ff9b4790307)

Should we keep it (and add to the other Adapters), for example like 
```
if (false === imagejpeg($collage, $localFile, 60) || filesize($localFile) === 0) {
```
?

Reply:
@kingjia90 I think the filesize is only be 0 when imagejpeg() returns false, so the check is not necessary.

Reply:
ahh ok , thank you